sb_download/main.py

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import sevenbridges as sbg
+import os
+import argparse
+
+os.environ['SB_API_ENDPOINT'] = 'https://cgc-api.sbgenomics.com/v2'
```

Comment:
Would this be a bit better? This way user will be able to set it other values if needed.
`os.environ['SB_API_ENDPOINT'] = 'https://cgc-api.sbgenomics.com/v2' if 'SB_API_ENDPOINT' not in os.environ`